DailySelfie/app/src/main/java/com/teamagam/dailyselfie/MainActivity.java

```diff
@@ -29,18 +26,30 @@
 
 public class MainActivity extends AppCompatActivity {
 
-    private final int DAILY_NOTIFICATION_ID = 0;
-
     private static final int REQUEST_TAKE_PHOTO = 1;
+    private static final String PREFERENCE_FIRST_RUN = "first_run";
 
     private PictureAdapter mPictureAdapter;
     private RecyclerView mRecyclerView;
 
-
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_main);
+        setAlarmInFirstRun();
```

Comment:
I'd place this method in the application class so it wont get executed every time the main activity builds itself.
The application class will only be executes once, like this code should be.